lib/taglet.ex

```diff
@@ -195,11 +199,12 @@ defmodule Taglet do
 
   You can pass a simple tag or a list of tags.
   """
-  @spec tagged_with(tags, module, context) :: list
-  def tagged_with(tags, model, context \\ "tags")
-  def tagged_with(tag, model, context) when is_bitstring(tag), do: tagged_with([tag], model, context)
-  def tagged_with(tags, model, context) do
-    do_tags_search(model, tags, context) |> repo().all
+  @spec tagged_with(tags, module, context, opts) :: list
+  def tagged_with(tags, model, context \\ "tags", opts \\ [prefix: nil])
```

Comment:
The opts default value should be an empty list. Fetching a non-existing key from a keyword list already produces an `nil` value by default.